app/transport/rest/user.go

```diff
@@ -60,3 +70,41 @@ func (uh UserHandler) CreateUser() http.Handler {
 		uh.logger.Debugw("User successfully created", "ID", id)
 	})
 }
+
+// GetUserByID will handle user search.
+func (uh UserHandler) GetUserByID() http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		id := mux.Vars(req)["id"]
+
+		if _, err := uuid.Parse(id); err != nil {
+			uh.logger.Warnw("Invalid request", "ID", id)
```

Comment:
Which request?.. A mystery.

Reply:
fixed

Reply:
It's more a comment for later but just imagine how this log will look like and whether it will say you anything if you read it in the logs.